src/Subscription.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Razorpay\Api;
+
+class Subscription extends Entity
+{
+    public function create($attributes = array())
+    {
+        return parent::create($attributes);
+    }
+
+    public function fetch($id)
+    {
+        return parent::fetch($id);
+    }
+
+    public function all($options = array())
+    {
+        return parent::all($options);
+    }
+
+    public function cancel()
+    {
+        $relativeUrl = $this->getEntityUrl() . $this->id . '/cancel';
+
+        $data = [
+            'cancel_at_cycle_end' => 1
```

Comment:
@StSarc Is this the default behaviour we want in the SDK?

Reply:
No. Default  behavior should be immediate